integration_tests/mockApis/keyworkerApi.ts

```diff
@@ -254,6 +252,8 @@ export default {
   stubKeyworkerApiStatusFail: () => stubKeyworkerApiStatusFail(),
   stubKeyworkerApiStats2025,
   stubKeyworkerApiStats2024,
-  stubKeyworkerMigrationStatus,
+  stubKeyworkerMigrationStatus: () => stubKeyworkerPrisonConfig(true, true),
```

Comment:
This stub is either not needed or should be renamed to stubEnabledPrisonWithHighComplex